cli/src/commands/init.rs

```diff
@@ -146,3 +113,67 @@ fn print_trackable_remote_branches(ui: &Ui, view: &View) -> io::Result<()> {
     )?;
     Ok(())
 }
+
+// TODO(essiene): Move to cli/src/commands/git.rs for `jj git init`
+fn git_init(
+    ui: &mut Ui,
+    command: &CommandHelper,
+    workspace_root: &Path,
+    use_git_backend: bool,
```

Comment:
Does this function needs `use_git_backend` at all?
When this become `cmd_git_init()`, there won't be `--git` flag anymore.

Reply:
d'oh! Thanks for the catch. Removed and the code is simpler.